docs/docusaurus/versioned_docs/version-1.5.0/orc8r/upgrade_1_5.md

```diff
@@ -2,22 +2,129 @@
 
-Run on a machine with access to the kubernetes cluster you are using to run Magma.
+As a pre-requisite task to orc8r upgrade to 1.5, the NMS DB migration needs to
+be completed. This migration copies all NMS data to the same database housing
+orc8r data.
+These tables are named
+`Users, Organizations, FeatureFlags, AuditLogEntry, SequelizeMeta`
+A single script is provided to perform this migration for you, and should
+tell you whether or not it was successful. After migration, the tables listed
+above should be present in the orc8r database.
+
+Run on a machine with access to the kubernetes cluster you are using to run
+Magma.
 
 *Paste in shell prompt:*
 
 `wget https://raw.githubusercontent.com/magma/magma/master/nms/app/packages/magmalte/scripts/fuji-upgrade/pre-upgrade-migration.sh && chmod +x pre-upgrade-migration.sh && ./pre-upgrade-migration.sh`
+
+*Terraform main.tf Sanitization*
+
+References to NMS DB need to be removed from the main.tf file.
+Further, the dialect for the orc8r DB needs to be specified.
+Please refer to the example main.tf located [here](https://github.com/magma/magma/blob/master/orc8r/cloud/deploy/terraform/orc8r-helm-aws/examples/basic/main.tf).
+Common dialects will be `mysql`, `postgres`, and `mariadb`.
+See [here](https://sequelize.org/v5/manual/dialects.html) for a list.
+
+## Upgrade Terraform modules
+
+Set the `source` values for each of the Orchestrator modules in your root
+Terraform module to point to this release's modules and bump chart and
+container versions
+
+```hcl-terraform
+# This will likely be found in main.tf
+
+module orc8r {
+  source = "github.com/magma/magma//orc8r/cloud/deploy/terraform/orc8r-aws?ref=v1.5"
+  # ...
+  cluster_version = "1.17"   # set to Kubernetes version found above. 1.17 is used as an example here
+}
+
+module orc8r-app {
+  source = "github.com/magma/magma//orc8r/cloud/deploy/terraform/orc8r-helm-aws?ref=v1.5"
+  # ...
+  orc8r_chart_version   = "1.5.20"
+  orc8r_tag             = "MAGMA_TAG"  # from build step, e.g. v1.5.0
+  orc8r_deployment_type = "fwa"        # valid options: ["fwa", "federated_fwa", "all"]
+}
+```
+
+Set `cluster_version` to the Kubernetes version found during the
+`Prerequisites` section. Bump your chart version to `1.5.20` and `orc8r_tag` to
+the semver tag you published your new Orchestrator container images as.
+You also need to set the `orc8r_deployment_type` variable to the deployment
+type that you intend to deploy. This type sets which orc8r modules will run.
+
+Then, prepare Terraform for the upgrade
+
+```bash
+terraform init --upgrade
+terraform refresh
+```
+
+## Terraform apply
+
+Terraform the upgrade.
+
+Apply the upgrade
+
+```bash
+terraform apply     # Check this output VERY carefully!
+```
+
+After the Terraform command completes, all application components should be
+successfully upgraded.
+
+
+## Post-Upgrade: Sync Pre-defined Alerts
+
+In v1.5, a new pre-defined alert has been added to raise a critical alert when
```

Comment:
New alert have been added to set of predefined alerts in 1.5. "Duplicate request alert" is raised as a critical alert, when we receive more than 100 alerts within last 5 minutes. In case you would like to customize this alert, refer to alerts guide.